tests/end_to_end/test_netcat/data/playbooks/generate_events.yaml

```diff
@@ -0,0 +1,34 @@
+- name: Truncate files
+  hosts: wazuh-manager
+  tasks:
+
+    - name: Truncate file alert.json
+      shell: echo "" > /var/ossec/logs/alerts/alerts.json
+      become: true
+
+- name: Generate events
+  hosts: wazuh-agent
+  ignore_errors: true
+  tasks:
+    - name: "{{ event_description_netcat }}"
+      command: "{{ command_netcat }}"
+      timeout: 30
+
+    - name: Wait for alert
+      wait_for:
+        timeout: 5
+
+    - name: Kill netcat
+      become: true
+      command: pkill -fx 'nc -l 8000'
+
+- name: Get logs
+  hosts: all
```

Comment:
This is necessary to run in all hosts?

Reply:
Done in d50708de21c0bf6e595247a72ad545c1127033e7